.flake8

```diff
@@ -3,4 +3,4 @@ ignore =
   E501, # line too long
   E241, # space after comma (ignored for list in gen-s-parser.py)
   W504  # line break after binary operator
-exclude = ./test/emscripten,./test/spec,./test/wasm-install
+exclude = ./test/emscripten,./test/spec,./test/wasm-install,./test/lit
```

Comment:
Why?

Reply:
flake8 complains about `config` being used before it is defined in the lit config and similar non-problems.

Reply:
How about `test/lib/*config.py` or something like that then?    Or will this directory be otherwise free of python code so it doesn't matter?

Reply:
Yes, this directory will otherwise be free of Python code.